scripts/azure-windows.yml

```diff
@@ -12,6 +12,35 @@ steps:
     fi
 
 - powershell: |
+
+    $env:Path += ";c:\msys64\usr\bin"
+
+    echo "what patches are here?"
+    dir .\cmake\inputs\patches\ep_azuresdk
+    
+    function deleteCR([string] $path, [string]$file){
+      write-host "path: $path"
+      write-host "file: $file"
+      $infile = Join-Path $path $file
+      write-host "infile: $infile"
+      $outname = $file + ".nocr"
+      write-host "outname: $outname"
+      $outfile = (Join-Path $path $outname )
+      write-host "outfile: $outfile"
+      cmd /c "c:\msys64\usr\bin\tr -d '\r' < $infile > $outfile"
+      if (Test-Path $outfile) {
+        dir $path
+        remove-item -path $infile 
+        ren $outfile $file 
+        dir $path
+      }
+    }
+    
+    deleteCR "./cmake/inputs/patches/ep_azuresdk" "azure-storage-lite-base64.patch"
```

Comment:
Can you clarify why this is necessary?

Reply:
I guess the problem might be patchfiles with a different line-ending than the rest of the repository after checkout, so they don't apply? It seems like something else must be going wrong here, and I would prefer to avoid this `deleteCR` function because it will be kind of confusing for the next person editing this file in six months. There are a couple options:
- use `git --ignore-whitespace`
- use `.gitattributes` in the patches folder to ensure that the patches are checked out with the correct ending

Reply:
Regarding .gitattributes, I did not find it used anywhere else within tiledb, so did not try it.
Regarding --ignore-whitespace, I seem to recall trying it and the patch still failed, but will revisit again